### ISSUE TITLE
Scheduled pgbackrest task failed to take backups

### ISSUE THREAD

**Issue by PowerStateFailure** [CrunchyData/postgres-operator#569]
**Describe the bug**
Scheduled task from `pgo schedule --schedule-type="pgbackrest"` can not take backups. Here's logs from crunchy-scheduler:

```
time="2019-01-30 09:52:00" level=info msg="Running pgBackRest backup" backupType=diff container=database deployment= label="pg-cluster=mycluster,service-name=mycluster" namespace=default
time="2019-01-30 09:52:00" level=error msg="Failed execing into container" backupType=diff container=database deployment= error="command terminated with exit code 72" label="pg-cluster=mycluster,service-name=mycluster" namespace=default stderr="WARN: option repo1-retention-full is not set, the repository may run out of space\n      HINT: to retain full backups indefinitely (without warning), set option 'repo1-retention-full' to the maximum.\nERROR [072]: : backup command must be run on the repository host\n" stdout="pg_pid=63\n"
```

By the way, `pgo backup` works fine, I can see successed manual backups via `pgo show backup`.

**To Reproduce**
1. Install postgres-operator v3.5.0 via helm (from ./chart subdir because of incorrect release archive - see #567 )
2. `pgo create cluster mycluster --pgbackrest`
3. `pgo create schedule mycluster --schedule="@every 1m" --schedule-type="pgbackrest" --pgbackrest-backup-type="diff"`
4. See error in `crunchy-scheduler` log

**Expected behavior**
I expect scheduled backups to work such way.

**Screenshots**

**Please tell us about your environment:**

* Operating System: Win7
* Where is this running ( Local, Cloud Provider):
* Storage being used (NFS, Hostpath, Gluster, etc): Hostpath (PVC)
* Container Image Tag:
* PostgreSQL Version: 10.6 (but crunchy-scheduler has version 11.0)
* Platform (Docker, Kubernetes, OpenShift): Minikube
* Platform Version: 11.1

**Additional context**

**Comment by jasonodonnell** [comment]
The helm chart for installing operator is installing the old scheduler (`crunchy scheduler`, should be `pgo-scheduler`).

Will fix this, thanks for the report

**Comment by jasonodonnell** [comment]
Fixed in #576, thanks!

**Comment by PowerStateFailure** [comment]
I've tested your changes, and now scheduled backup works properly, thanks!
